### PR TITLE
Adding access to the SplitNames functionality

### PR DESCRIPTION
**Pull request by HJewkes**
Jira Ticket: [SPLIT-2315](https://splitio.atlassian.net/browse/SPLIT-2315)

### CODE REVIEW

File: src/main/java/io/split/client/SplitManager.java [hunk 1]
```diff
@@ -22,4 +22,11 @@
      * @return SplitView or null
      */
     SplitView split(String featureName);
+
+    /**
+     * Returns the names of features (or Split) registered with the SDK.
```
Comment (SenhorCastor) [inline src/main/java/io/split/client/SplitManager.java]: typo Splits

Comment (HJewkes) [inline src/main/java/io/split/client/SplitManager.java]: Fixed

File: src/main/java/io/split/engine/matchers/LessThanOrEqualToMatcher.java [hunk 2]
```diff
@@ -53,7 +53,7 @@ public String toString() {
     @Override
     public int hashCode() {
         int result = 17;
-        result = 31 * result + Long.hashCode(_compareTo);
+        result = 31 * result + (int)(_compareTo ^ (_compareTo >>> 32));
```
Comment (patricioe) [inline src/main/java/io/split/engine/matchers/LessThanOrEqualToMatcher.java]: is this better than Long.hashCode?

Comment (HJewkes) [inline src/main/java/io/split/engine/matchers/LessThanOrEqualToMatcher.java]: According to @SenhorCastor it was a Java 7 Compatability change

**Pull request merged**